### DISCUSSION TITLE
lmdeploy serve api_server 可以把两个模型部署到同一个端口吗？

### DISCUSSION THREAD

**Discussion by thomZ1** [InternLM/lmdeploy#2053] [Q&A] [answered]
lmdeploy serve api_server 可以把两个模型部署到同一个端口吗？ 
`lmdeploy serve api_server ChatGLM/glm-4-9b-chat --model-name glm4  --server-port 8800  
lmdeploy serve api_server Qwen2-7B-Instruct --model-name qwen2  --server-port 8800  `
可以这样只用model name不一样，端口一样吗

**Comment by zhyncs** [comment, marked as answer]
Nope

**Reply by thomZ1** [reply]
😭

**Reply by shell-nlp** [reply]
https://github.com/shell-nlp/gpt_server 我的项目是支持的， 支持lmdeploy后端 @thomZ1
